src/modules/site-v2/templates/_includes/phenotype-database-table.html

```diff
@@ -291,15 +291,19 @@ <h5 id="offcanvas1Label"></h5>
                     tagsHTML += `<div class="mb-3"><a role="button" class="btn btn-link" id="tagLink-${j+1}" data-value="${tags[j]}">${tags[j].replaceAll('/', ' • ')}</a></div>`
                 }
             }
-            
+
```

Comment:
should this be a `partial` jinja template instead?

Reply:
As we're fetching data on the client-side I was only able to update html with Javascript. But I'll be happy to implement it with Jinja template if there's a way. Please let me know.